src/HPolygonOpt.jl

```diff
@@ -182,3 +182,39 @@ function σ(d::AbstractVector{N}, P::HPolygonOpt{N};
         end
     end
 end
+
+"""
+    translate(P::HPolygonOpt{N}, v::AbstractVector{N}; share::Bool=false
+             ) where {N<:Real}
+
+Translate (i.e., shift) an optimized polygon in constraint representation by a
+given vector.
+
+### Input
+
+- `P`     -- optimized polygon in constraint representation
+- `v`     -- translation vector
+- `share` -- (optional, default: `false`) flag for sharing unmodified parts of
+             the original set representation
+
+### Output
+
+A translated optimized polygon in constraint representation.
+
+### Notes
+
+The `a` vectors of the constraints are shared with the original constraints.
```

Comment:
The normal vectors of the constraints (vector `a` in `a⋅x ≤ b`) are shared with the original constraints if `share=true`.